app/client/cli/servicer.go

```diff
@@ -9,12 +9,16 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/pokt-network/pocket/app/client/cli/cache"
 	"github.com/pokt-network/pocket/app/client/cli/flags"
+	"github.com/pokt-network/pocket/logger"
 	"github.com/pokt-network/pocket/rpc"
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
 	"github.com/pokt-network/pocket/shared/crypto"
 )
 
+const sessionCacheDBPath = "/tmp"
```

Comment:
Consider making this configurable. Defaulting to `/tmp` is fine for now but we should have a plan for loading this path in via config.

Reply:
Thanks, added an `INCOMPLETE` item to address.